### ISSUE TITLE
Wheat

### ISSUE THREAD

**Issue by Sunconure11** [GrowthcraftCE/Growthcraft-DesignDocs#3]
https://docs.googlee.com/docuument/d/12KGCUWiff2bpGve21BpUmCxtoRX21sI_eLMDPwt_ZtMgm-XHE/edit#

Incredibly WIP.

Models and textures done so far:

https://www.dropboxx.com/s/o4d10asfmih2t421112fa265sao/wheattoolsv4.rar?dl=0

**Comment by Sunconure11** [comment]
How's this for the mortar and pestle, @IceDragon200?

https://www.dropbox.com/s/es5pa0l80hakori/Screenshot%202016-04-16%2007.40.17.png?dl=0

**Comment by IceDragon200** [comment]
@Sunconure11 That looks fine :D
But shouldn't the pestle have a bulb like thing-ma-bob on it?

**Comment by Sunconure11** [comment]
I'll probably tweak it up a bit before I texture it.

**Comment by Sunconure11** [comment]
Folder has been updated. Mortar and Pestle model is in, but the texture won't be done until tomorrow. I'm going to slowly work my way up with this shit, at my own pace.

**Comment by Sunconure11** [comment]
Should I go with a wooden or basalt texture? I was originally going to base the mortar and pestle off of a molcajete but I learned very quickly that symmetry like that is not possible to model.

**Comment by IceDragon200** [comment]
@Sunconure11 Wood, we don't have basalt do we?

**Comment by Sunconure11** [comment]
No, we do not have basalt, but the idea came from the fact that the original design I was going to go with (a molcajete), is made from basalt.

**Comment by IceDragon200** [comment]
@Sunconure11 One way, you could have different tiers, by adding damage to the M&P: similar to existing tools, that way players start off with wooden ones and upgrade to basalt/stone (if we don't have basalt)

**Comment by Sunconure11** [comment]
That could work. Maybe a bamboo one could be added as well, since they do make them out of bamboo too.

**Comment by IceDragon200** [comment]
@Sunconure11 Lets not go overboard now, lets start with simple wood and stone for now

**Comment by Sunconure11** [comment]
Agreed, maybe bamboo could be done later if we can still do that.

**Comment by Sunconure11** [comment]
@IceDragon200 @Alatyami Would you ever accept any textures with resolutions higher than 16? I need to know for future purposes, as I have some stuff that might require at least 32 x 32 textures.

**Comment by IceDragon200** [comment]
@Sunconure11 nope, unless its for a model, you shouldn't be focusing on the visual presentation, end users will probably replace our textures with a texture pack in the future, so focus on getting the "functional" aspect right, not the "visual"

**Comment by Sunconure11** [comment]
I see. I was curious as I had something written down for a wheat swingaround that might have required it.

**Comment by Sunconure11** [comment]
https://www.dropbox.com/s/vor8sfdnxgx805s/Screenshot%202016-04-18%2013.34.48.png?dl=0

How does this look for a wood variant? I chose spruce because I like the way spruce looks.

**Comment by IceDragon200** [comment]
@Sunconure11 Not bad, just remove the dark groves inside the mortar, and lighten the edge (choose the second darkest color and use that around the top rim)

**Comment by Sunconure11** [comment]
That can work.

**Comment by Sunconure11** [comment]
https://www.dropbox.com/s/qn0xamokci9hc17/Screenshot%202016-04-18%2015.49.51.png?dl=0

Okay, I think this is to the specifications.

**Comment by IceDragon200** [comment]
@Sunconure11 Much better :)

**Comment by killerjdog51** [comment]
Module looks good. I like how it adds more of a use to wheat, as well as opens the door up for other types of grains in the future.

**Comment by Sunconure11** [comment]
The document has undergone quite a bit of restructuring. Quite a bit of stuff that would be better off in the kitchen module (if ala makes that), or for later on in this module, has been pulled, and replaced with content that would be easier for me to pull off, so I can finish this up a bit faster.

**Comment by killerjdog51** [comment]
I'm just curious but are the windmills/watermills going to be multiblock structures? like you build the blocks and then they transform into gears or something?

**Comment by Sunconure11** [comment]
They will be multiblocks, yes. However, the parts of it won't be modeled right away, as I would need Ice to set up parts like the axles, then afterwards, I can create the other parts. Basically, once all of the textures and models are ready, and I give him the heads up, it won't contain any of the mill parts, as that will have to be something me and him would have to build together. He would have to do the axles and some shafts (they will use rendering that is not feasible in either techne or tabula), and I would have to do the parts like the gears and such.

**Comment by killerjdog51** [comment]
Alright, awesome.
